src/core/ComboBox/ComboBox.tsx

```diff
@@ -356,7 +385,15 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
             role='listbox'
             ref={menuRef}
           >
-            {menuItems}
+            {enableVirtualization ? (
+              <VirtualScroll
+                itemsLength={menuItems.length}
+                itemRenderer={(index) => menuItems[index] ?? <></>}
+                scrollToIndex={focusedIndex}
```

Comment:
There is a state update happening after the menu is closed
![image](https://user-images.githubusercontent.com/9084735/152825952-cd3d73e3-f473-4b21-8fd2-082d2e56591f.png)

I haven't investigated but I'm guessing it has to do with this focusedIndex passed into VirtualScroll

Reply:
As for now, cant find anyhow, where is the issue :/
Tried commenting all the useEffect and setStates, but could not get the result without error..Went thourgh VS, Combobox and Menu components.. Unless I still missed something

Reply:
Fixed 🎉